### PR TITLE
Support "size" and "timeout" on *_fuzz_test

### LINKED ISSUE #202: Add support for test size/timeout in `cc_fuzz_test`.
## Expected Behavior

Test success without warnings.

## Actual Behavior

```
ERROR: /src/workspace/c-toxcore/toxcore/BUILD.bazel:497:13: //c-toxcore/toxcore:group_moderation_fuzz_test_raw_: no such attribute 'size' in 'cc_binary' rule
```

## Steps to Reproduce the Problem

1. Add `cc_fuzz_test(... size = "small")`
1. `bazel build //...`
1. Profit. (or not)

## Specifications

- Version: v0.3.1
- Platform: Linux/x86_64, bazel 5.0.0.

### PR DESCRIPTION
**Pull request by fmeum**
Fixes #202

### CODE REVIEW

**Comment by fmeum** [comment]
@stefanbucur The CI failure is fixed by #205. I rebased this PR onto it.

**Review by stefanbucur — CHANGES_REQUESTED** [review]
Great, thank you!

File: fuzzing/private/fuzz_test.bzl [hunk 1]
```diff
@@ -164,7 +172,9 @@ def cc_fuzz_test(
         corpus: A list containing corpus files.
         dicts: A list containing dictionaries.
         engine: A label pointing to the fuzzing engine to use.
+        size: The size of the fuzzing regression test.
         tags: Tags set on the fuzzing regression test.
+        timeout: The timeout for the fuzzing regression test.
```
[the remark this answers is not in the document]
Comment (fmeum) [reply fuzzing/private/fuzz_test.bzl]: Added a link and a clarified the distinction between "fuzzing" and "regression test". Do you think this is clear enough now?

[the remark this answers is not in the document]
Comment (stefanbucur) [reply fuzzing/private/fuzz_test.bzl]: Looks great!

**Review by stefanbucur — APPROVED** [review]
Apologies for the delay!

**Pull request merged**